botocore/args.py

```diff
@@ -26,6 +26,13 @@
 import botocore.serialize
 from botocore.config import Config
 from botocore.endpoint import EndpointCreator
+
+# We have moved the PRIORITY_ORDERED_SUPPORTED_PROTOCOLS list out of this file
+# so we import it here to maintain backwards compatibility.  The redundant "as"
+# statement tells tools like Ruff not to remove it
```

Comment:
Couldn't you put a `# noqa` comment on the line to achieve this, and then wouldn't need to document this?

Reply:
`noqa` and the redundant import are effectively the same thing, right?

I wanted to keep the comment there to explain why the import is necessary and prevent people from removing it in the future.  Do you think it's better without it?